### DISCUSSION TITLE
uncrustify for c++

### DISCUSSION THREAD

**Discussion by HazemKurdy** [uncrustify/uncrustify#3862] [Q&A]
hey All,
I got a couple of question :
1- is there a method to have two or more configuration files (for example one for c and one for c++)
2- is there a way to add regular expression instead of file/folder name in IgnoreList
3- is there a way to disable the indenting of code line breaking '\'
4- can uncrustify run on python files?
for example:
#define a(x) \
if (x) \
   return x; \
thanks

**Comment by guy-maurel** [comment]
1. yes, 
2. no
3. yes
```
 # Specify the marker used in comments to disable processing of part of the
# file.
#
# Default:  *INDENT-OFF*
disable_processing_cmt          = " *INDENT-OFF*"      # string

# Specify the marker used in comments to (re)enable processing in a file.
#
# Default:  *INDENT-ON*
enable_processing_cmt           = " *INDENT-ON*"     # string
```
4. I never try
